blocks/library/latest-comments/index.js

```diff
@@ -0,0 +1,227 @@
+/**
+ * External dependencies
+ */
+import { keys, max } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from 'element';
```

Comment:
Note with #2172, these need to be updated to prefix the dependencies with `@wordpress/`. You will need to perform a rebase against the latest version of master and apply your changes:

```
git fetch origin
git rebase origin/master
```